app/Http/Controllers/FacultyController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Services\IndexFacultiesService;
+use Illuminate\Http\JsonResponse;
+
+class FacultyController extends Controller
+{
+    public function get(): JsonResponse
```

Comment:
Better name for this fuction is a getAll becouse you describe what you get from this fucktion